### PR TITLE
gofmt before tests

### PR DESCRIPTION
**Pull request by danielhochman**
also put travis on 1.7.5

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,8 +23,12 @@ fix_format:
 	script/docs_fix_format
 	go fmt $(shell glide nv)
 
+.PHONY: check_format
+check_format: docs_format
+	@gofmt -l $(shell glide nv | sed 's/\.\.\.//g') | tee /dev/stderr | read && echo "Files failed gofmt" && exit 1 || true
+
 .PHONY: compile
-compile: docs_format
```
Comment (danielhochman) [inline Makefile]: was this needed @junr03?

File: Makefile [hunk 2]
```diff
@@ -22,8 +23,12 @@ fix_format:
...
+compile:
```
Comment (junr03) [inline Makefile]: yep I think that is better, no need to format the docs on the compile target.

File: Makefile (hunk 1 above)
Comment (junr03) [inline Makefile]: not needed, that was legacy. I think that putting that in the check_format target is better, and makes more sense.

File: src/redis/cache.go [hunk 3]
```diff
@@ -12,8 +12,11 @@ type TimeSource interface {
 	UnixNow() int64
 }
 
+// Interface for a rand Source for expiration jitter
```
Comment (junr03) [inline src/redis/cache.go]: missing `.`

Comment (junr03) [inline src/redis/cache.go]: and below

File: src/redis/cache_impl.go [hunk 4]
```diff
@@ -189,7 +189,6 @@ func (this *timeSourceImpl) UnixNow() int64 {
 }
 
 // rand for jitter
```
Comment (junr03) [inline src/redis/cache_impl.go]: missing `.`

**Pull request merged**